<launch>

	<node pkg="telicam_driver" type="ros_cam_driver_1" name="ros_cam_driver_1" output="screen">
 		
		<!--param name="ros_node_name" value="teli_0"/-->
		<param name="camera_index" value="1"/>
		<param name="message_name" value="img_1"/>
		<param name="image_frame_name" value="camera_1"/>
		
		<param name="ifTrigger" value="true"/>
		<param name="exposure_value" value="10000"/>
		<param name="gain_value" value="10"/>

 	</node>
	
	<!--node pkg="rviz" type="rviz" name="rviz" args="-d $(find telicam_driver)/cfg/image.rviz" output="screen"/-->

</launch>
